model/account.rb

```diff
@@ -15,6 +15,10 @@ class Account < Sequel::Model(:accounts)
 
   alias_method :admin_label, :email
 
+  def provider_names
+    identities.map(&:provider)
```

Comment:
Up to you, but maybe:

```suggestion
    identities.map(&:provider).join(", ")
```